scripts/hdiff.sh

```diff
@@ -0,0 +1,8 @@
+base=$(basename $1 .test)
+
+out=$(find . -name ${base}.out)
+exp=$(find . -name ${base}.regexpect)
+
+ORION_DIFF=${ORION_DIFF:-tkdiff}
+
+$ORION_DIFF $out $exp
```

Comment:
Other places use CB_DIFF_TOOL (e.g. viewDiff.sh). I would suggest to use the same name for the env variable.